src/main/java/org/zalando/nakadi/domain/SubscriptionListWrapper.java

```diff
@@ -0,0 +1,19 @@
+package org.zalando.nakadi.domain;
+
+import javax.annotation.concurrent.Immutable;
+import java.util.Collections;
+import java.util.List;
+
+@Immutable
+public class SubscriptionListWrapper {
```

Comment:
Maybe this pagination wrapper could be more generic upfront. We might use such wrapper for other entities.